test/exq_scheduler_time_test.exs

```diff
@@ -45,6 +45,8 @@ defmodule ExqSchedulerTimeTest do
 
     jobs = get_jobs("TimeWorker", "TimeQ")
 
+    assert length(jobs) > 0
```

Comment:
any specific reason for this assert? as this fails locally for me as well.

Reply:
I added it because the following assertion will pass even if there are no jobs, which seems like it would be misleading at best

Reply:
I think that this test has been broken for quite some time, I can replicate the same failure on master just by adding this assertion

Reply:
It looks like that test is the only thing failing the build at this point, so I'm going to leave the assertion in, but tag the test as `skip: true`

Reply:
I have adjusted the sleep time to fix this issue